src/modules/core/components/avatar/avatar.tsx

```diff
@@ -0,0 +1,82 @@
+import { type FC, useMemo } from 'react';
+import clsx from 'clsx';
+import Image from 'next/image';
+
+import type { AvatarProps } from './avatar.interface';
+import styles from './avatar.module.scss';
+
+export const Avatar: FC<AvatarProps> = ({
+  url,
+  size = 'small',
```

Comment:
I don't really like restricting Avatar to only three predefined sizes. In my opinion, it would be better to set size ourselves with of course some default value.